src/api/.keystone/config.js

```diff
@@ -548,6 +566,32 @@ var verifyAccountEmailAddress = async (root, variables, context) => {
 };
 var verify_account_email_address_default = verifyAccountEmailAddress;
 
+// custom-resolvers/send-email-confirm-email-address.ts
+var sendEmailConfirmEmailAddress = async (root, variables, context) => {
```

Comment:
const/let?

Reply:
Same as above, anything insdie of `.keystone` directory is auto generated